stronk/models/workout.py

```diff
@@ -37,6 +41,8 @@ def update(self, attrs):
             self.description = attrs.get('description')
         if attrs.get('projected_time'):
             self.projected_time = attrs.get('projected_time')
+        if attrs.get("scheduled_time"):
+            self.scheduled_time = attrs.get("scheduled_time")
```

Comment:
This doesn't do any casting from `str` to `datetime` but the column expects a `datetime` object. We don't need to necessarily do it here, but just something to point out.

Reply:
Yee `scheduled_time` should be a datetime object. I did the parsing in the calling function bc I wasn't sure if it was a good idea to handle parsing logic inside of the mode (ie. model should only be aware of the `datetime` and not its string rep)